pkg/controllers/multicluster/cloudmap_controller.go

```diff
@@ -180,24 +180,6 @@ func (r *CloudMapReconciler) reconcileService(ctx context.Context, svc *model.Se
 		if err = r.updateEndpointSlices(ctx, svcImport, endpoints, derivedService, clusterId); err != nil {
 			return err
 		}
-
-		derivedServices = append(derivedServices, derivedService)
-	}
-
-	// remove any existing derived services that do not have any endpoints in cloud map
-	existingDerivedServices := &v1.ServiceList{}
```

Comment:
Is this code getting removed?

Reply:
should not be

Reply:
really weird - shows as added in my IDE - let me try again